web-jehlomat/src/utils/url.ts

```diff
@@ -0,0 +1,8 @@
+export function convertSearchParamsToString(params: Record<string, string>): string | undefined {
+  const searchStringParams = new URLSearchParams(params).toString();
+    return searchStringParams === "" ? undefined : `?${searchStringParams}`;
+}
+
+export function convertSearchParamsToMap(params: string): URLSearchParams {
```

Comment:
Please consider renaming, converting from `URLStringParams` > `string` is done by `convertSearchParamsToString` but opposite direction uses similar name `convertSearchParamsToMap`; bot are using `convertSearchParamsXX` at the beginning. Maybe `convertStringToSearchParams`?